content/geoip/docs/databases/anonymous-ip.md

```diff
@@ -0,0 +1,165 @@
+| --------------------- | ---------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
+| .NET (C#)             | [GitHub - Database Usage](https://github.com/maxmind/GeoIP2-dotnet#database-usage) | [Anonymous IP Example](https://github.com/maxmind/GeoIP2-dotnet#anonymous-ip-database)       |
+| Java                  | [GitHub - Database Usage](https://github.com/maxmind/GeoIP2-java#database-usage)   | [Anonymous IP Example](https://github.com/maxmind/GeoIP2-java#anonymous-ip)                  |
+| Node.js               | [GitHub - Database Usage](https://github.com/maxmind/GeoIP2-node#database-usage)   | [Anonymous IP Example](https://github.com/maxmind/GeoIP2-node#anonymous-ip-database-example) |
+| PHP                   | [GitHub - Database Reader](https://github.com/maxmind/GeoIP2-php#database-reader)  | [Anonymous IP Example](https://github.com/maxmind/GeoIP2-php#anonymous-ip-example)           |
+| Python                | [GitHub - Database Usage](https://github.com/maxmind/GeoIP2-python#database-usage) | [Anonymous IP Example](https://github.com/maxmind/GeoIP2-python#anonymous-ip-database)       |
+| Ruby                  | [GitHub - Database Reader](https://github.com/maxmind/GeoIP2-ruby#database-reader) | [Anonymous IP Example](https://github.com/maxmind/GeoIP2-ruby#anonymous-ip-example)          |
+{{</ table >}}
+
+## CSV Database
+
+{{< snippet "snippets/csv-databases-intro.md" >}}
+
+{{< zip-file-content product-name="GeoIP2-Anonymous-IP" exclude-table="false" >}}
+
+{{< snippet "snippets/file-format.md" >}}
+
+### Blocks Files
+
+There are two CSV files for network blocks, one each for IPv4 and IPv6 blocks.
+These are named `GeoIP2-Anonymous-IP-Blocks-IPv4.csv` and
+`GeoIP2-Anonymous-IP-Blocks-IPv6.csv` respectively.
+
+{{< rawhtml >}}
+<div class="table">
+  <table>
+    <tbody>
+      <tr>
+          <th>Name</th>
+          <th>Type</th>
+          <th>Description</th>
+      </tr>
+
+      <tr>
+          <td>network</td>
+          <td>IP network as a string</td>
+          <td>
+            This is the IPv4 or IPv6 network in CIDR format such as
+            “2.21.92.0/29” or “2001:4b0::/80”. We offer a utility to convert
+            this column to start/end IPs or start/end integers. See <a
+            href="#conversion-utility">the conversion utility section</a> for
+            details.
+          </td>
+      </tr>
+
+      <tr>
+          <td>is_anonymous</td>
+          <td>boolean</td>
+          <td>
+            1 if the IP address belongs to any sort of anonymous network. Blank if not.
+            <br />
+            <a href="https://support.maxmind.com/hc/en-us/articles/4408208507163-Anonymizer-and-Proxy-Data#h_01FN9BBGV3ZG3TC0357Q9Y07C6" target="_blank" rel="nofollow noopener noreferrer">Learn more about anonymizer and proxy detection on our Knowledge Base.</a>
```

Comment:
Per https://www.pivotaltracker.com/story/show/187767361 should prevent links from opening to a new window?